main/fakeroot/APKBUILD

```diff
@@ -13,8 +13,7 @@ makedepends_build="libtool autoconf automake po4a"
 makedepends_host="libcap-dev acl-dev linux-headers"
 makedepends="$makedepends_build $makedepends_host"
 subpackages="$pkgname-doc"
-source="http://ftp.debian.org/debian/pool/main/f/$pkgname/${pkgname}_${pkgver}.orig.tar.bz2
-	busybox-getopt.patch
```

Comment:
you no longer apply busybox-getopt.patch but the patch was not removed. Has the patch been applied upstream? Is the handling of args with spaces fixed upstream? (see https://bugs.alpinelinux.org/issues/3500)

Reply:
`busybox-getopt.patch` removed. `fakeroot 1.23` runs as expected without util-linux.
Tested and confirmed against the script provided [here](https://bugs.alpinelinux.org/issues/3500).